### PR TITLE
Rename header of the gzip paragraph

### PR DESCRIPTION
**Pull request by mariusz-blaszczak**
### Summary

IMO the previous header (_Serving GZipped version of assets_) was misleading, because it was suggesting that the config will enable us serving the gzipped version of assets out of the box, which is not true, because we need to configure it on a server, e.g. nginx.

In fact, the paragraph is explaining how to enable/disable gzipping assets in the config and I believe that my change will be more straightforward for readers.

### CODE REVIEW

File: guides/source/asset_pipeline.md [hunk 1]
```diff
@@ -1101,7 +1101,7 @@ Windows you have a JavaScript runtime installed in your operating system.
 
 
 
-### Serving GZipped version of assets
+### Generating GZipped version of assets
```
Comment (gmcgibbon) [inline guides/source/asset_pipeline.md]: I think calling it `GZipping your assets` might be best, and then adding some text below the code block addressing how to actually serve gzipped assets. Something like `Refer to your web server's documentation for instructions on how to serve GZipped assets.` maybe?

Comment (blaszczakphoto) [reply guides/source/asset_pipeline.md]: Yes, this sounds good to me. I will make a change and push it today :)

**Review by gmcgibbon — COMMENTED** [review]
LGTM, can you squash your commits? Thanks!

**Comment by blaszczakphoto** [comment]
@gmcgibbon Just squashed the commits :)

**Review by gmcgibbon — APPROVED** [review]

**Pull request merged**